### PR TITLE
Change README from default Jekyll Clean

### PR DESCRIPTION
**Pull request by jbmagination**
Feel free to edit at will, however the credit for the theme must be included as stated in the original theme repository:

"The content of this theme is distributed and licensed under a Creative Commons Attribution 4.0 License:
`
This license lets others distribute, remix, tweak, and build upon your work,
even commercially, as long as they credit you for the original creation. This
is the most accommodating of licenses offered. Recommended for maximum
dissemination and use of licensed materials.
`
In other words: you can do anything you want with this theme on any site, just please provide a link to the original theme on github so I get credit for the original design. Beyond that, have at it!"

### CODE REVIEW

**Comment by jbmagination** [comment]
Hey, I just noticed that I'm the only person who's submitted a pull request, ever. Interesting.
